packages/manager/modules/sms/src/sms/batches/create/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,90 @@
+  "sms_message_commercial_description": "Pour des raisons légales, un SMS à caractère commercial ne sera envoyé qu’<strong>entre 8 h et 20 h, du lundi au samedi</strong>.",
+  "sms_no_commercial_clause_label": "Je certifie sur l’honneur que le SMS n’est pas à caractère commercial.",
+  "sms_programming": "Programmation de la campagne",
+  "sms_programming_date": "Date :",
+  "sms_programming_time": "Heure :",
+  "sms_options_type": "Type de SMS",
+  "sms_batches_create_send": "Envoyer",
+  "sms_batches_create_cancel": "Annuler",
+  "sms_credit_balance": "Solde",
+  "sms_credit_remaining_sms": "SMS restants",
+  "sms_credit_manage_credits": "Gérer vos crédits SMS OVHCloud",
+  "sms_tips_title": "Astuces",
+  "sms_tip_compose": "Composition d’un SMS",
+  "sms_tip_size": "Taille d’un SMS",
+  "sms_sender": "Expéditeur",
+  "sms_sender_manage_senders": "Gérer les expéditeurs",
+  "sms_sender_option_short_number": "Numéro permettant la réponse (France uniquement)",
+  "sms_sender_optgroup_label_virtual": "Expéditeurs permettant les réponses",
+  "sms_sender_optgroup_label_alpha": "Expéditeurs alphanumériques",
+  "sms_sender_optgroup_label_other": "Autres expéditeurs",
+  "sms_batches_create_receivers": "Destinataires",
+  "sms_phonebook_manage": "Carnets d’adresses",
+  "sms_receivers_add_list": "Liste des destinataires",
+  "sms_receivers_placeholder": "Numéro du destinataire (exemple : +33601020304)",
+  "sms_add_receivers_choice_title": "Ajouter une liste de destinataires",
+  "sms_add_receivers_choice_select_lists": "Sélectionnez les fichiers contenant les destinataires :",
+  "sms_add_receivers_choice_contact": "contact",
+  "sms_add_receivers_choice_contacts": "contacts",
+  "sms_receivers_list_selected": "liste sélectionnée",
+  "sms_receivers_list_selected_many": "listes sélectionnées",
+  "sms_contacts_selected": "contact sélectionné",
+  "sms_contacts_selected_many": "contacts sélectionnés",
+  "sms_add_contacts_choice_title": "Ajouter des contacts",
+  "sms_add_contacts_choice_info": "Pour envoyer des campagnes de SMS, utilisez de préférence un fichier CSV de destinataires.",
+  "sms_add_contacts_select_label": "Sélectionnez votre carnet d’adresses :",
+  "sms_add_contacts_filter_select_label": "Filtre :",
+  "sms_add_contacts_filter_select_all": "Tous",
+  "sms_add_contacts_filter_select_homeMobile": "Mobile (domicile)",
+  "sms_add_contacts_filter_select_homePhone": "Fixe (domicile)",
+  "sms_add_contacts_filter_select_workMobile": "Mobile (professionnel)",
+  "sms_add_contacts_filter_select_workPhone": "Fixe (professionnel)",
+  "sms_add_contacts_contact_surname": "Nom",
+  "sms_add_contacts_contact_name": "Prénom",
+  "sms_add_contacts_contact_phonenumber": "Numéro",
+  "sms_add_contacts_contact_type": "Type",
+  "sms_add_contacts_contact_homeMobile": "Mobile <span class=\"text-small\">(domicile)</span>",
```

Comment:
i think you should split this one in two translations "Mobile" and "domicile"
then in the template do something like
```
<span translate="..."></span>
<span class="text-small">
(<span translate="..."></span>)
</span>
```
it's not perfect but at least you don't have html tag and bootstrap class in the translation

Reply:
done